src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.ParserService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportMakerService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.service.impl.ParseServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportMakerServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        ReaderService reader = new ReaderServiceImpl();
+        ParserService parser = new ParseServiceImpl();
+        FruitShopService fruitShopService = new FruitShopServiceImpl();
+        ReportMakerService reportMakerService = new ReportMakerServiceImpl();
+        WriterService writer = new WriterServiceImpl();

```

Comment:
can we initialize services as class-level fields?

Reply:
yes, we can do it and I did it now)